tests/sdk/test_config.py

```diff
@@ -32,6 +34,49 @@ def test_add_load_profile(tmpdir):
     assert profile_1 == profile_2
 
 
+def test_add_load_profile_from_file(tmpdir):
+    path = tmpdir / 'substra.cfg'
+    conf = {
+       "node-1": {
+           "auth": {
+               "username": "node-1"
+           },
+           "insecure": False,
+           "url": "http://substra-backend.node-1.com",
+           "version": "0.0"
+       },
+    }
+
+    path.write_text(json.dumps(conf), "UTF-8")
+    keyring.set_password('node-1', conf['node-1']['auth']['username'], 'foo')
+    manager = configuration.Manager(str(path))
+    profile = manager.load_profile('node-1')
+
+    assert conf['node-1']['auth']['username'] == profile['auth']['username']
```

Comment:
Instead of checking just one property, I think you should be able to test all of them with something like 
```python
assert conf['node-1'] == profile
```

Reply:
You're right, it could be easier!